content/guides/references/migration-guide.md

```diff
@@ -83,6 +83,155 @@ on('<event>', (arg1, arg2) => {
 
 CONTENT_TBD
 
+### Attaching Files
+
+Attaching files to input elements or dropping them over the page is available
+in Cypress 10.0. Read the [`.attachFile()` API docs](/api/commands/attachfile)
+for more information on how this works and how to use it.
+
+The
+[`cypress-file-upload`](https://github.com/abramenal/cypress-file-upload)
+plugin has been deprecated in favor of `.attachFile()` built into Cypress.
```

Comment:
We should explicitly state that the cypress-file-upload is incompatible with 10.0, and will no longer work.

Reply:
Done.